libmscore/score.h

```diff
@@ -376,6 +376,7 @@ class Score : public QObject, public ScoreElement {
       QString _layerTagComments[32];
       QList<Layer> _layer;
       int _currentLayer { 0 };
+      int _isAnnotation { 0 };
```

Comment:
No, I'm just setting a flag for the paintevent to check if a box has to be drawn. It should rather have been called "hasAnnotation". See my comment below.